guides/common/modules/proc_enabling-structured-apt-content.adoc

```diff
@@ -0,0 +1,52 @@
+[id="enabling-structured-apt-content"]
+= Enabling structured APT content
+
+You can enable structured APT content on your {Project} to use the same Deb repository structure as the remote repositories they are synchronized from.
+
+.Prerequisites
+* The host is registered to {ProjectName}.
```

Comment:
What host? I thought we're performing this procedure on ProjectServer. This confuses me.

Reply:
It's a prerequisite for hosts that consume structured APT content from Foreman+Katello. -> Moved.